README.md

```diff
@@ -121,8 +121,8 @@ addition to `--profile ci` in the above command.
 
 ## The Vision
 
-We are not going to stop at replicating ChatGPT. We want to build the assistant
-of the future, able to not only write email and cover letters, but do meaningful
+We are not going to stop replicating ChatGPT. We want to build the assistant of
```

Comment:
Can we revert these changes? "stop at replicating" is correct here, not "stop replicating" and the original text is fine

Reply:
Yes, I can do that.. That came from my auto-correct..